src/main/java/com/philderbeast/prisonpicks/Util.java

```diff
@@ -104,14 +122,22 @@ static boolean canBuild(Player player, Location location)
         WorldGuardPlugin wg = getWorldGuard(); 
         if (wg != null)
         {
-            World world = location.getWorld(); 
-            ApplicableRegionSet regions = wg.getRegionManager(world).getApplicableRegions(BukkitUtil.toVector(location)); 
+            LocalPlayer localPlayer = WorldGuardPlugin.inst().wrapPlayer(player);
```

Comment:
why is this changed from using the custom flag?

Reply:
worldguard wont register custom flag, temp work around.